android-plugin/src/main/java/io/ecocode/xml/XmlRulesDefinition.java

```diff
@@ -19,16 +19,21 @@
  */
 package io.ecocode.xml;
 
+import org.sonar.api.SonarEdition;
+import org.sonar.api.SonarQubeSide;
+import org.sonar.api.SonarRuntime;
+import org.sonar.api.internal.SonarRuntimeImpl;
 import org.sonar.api.server.rule.RulesDefinition;
+import org.sonar.api.utils.Version;
 import org.sonarsource.analyzer.commons.RuleMetadataLoader;
 
 public final class XmlRulesDefinition implements RulesDefinition {
 
     @Override
     public void define(Context context) {
         NewRepository repository = context.createRepository(Xml.REPOSITORY_KEY, Xml.KEY).setName(Xml.REPOSITORY_NAME);
-
-        RuleMetadataLoader ruleMetadataLoader = new RuleMetadataLoader(Xml.XML_RESOURCE_PATH, Xml.PROFILE_PATH);
+        SonarRuntime sonarRuntime = SonarRuntimeImpl.forSonarQube(Version.create(9, 7), SonarQubeSide.SCANNER, SonarEdition.DEVELOPER);
```

Comment:
please, modify with Sonar version 9.8 instead of 9.7. same review note for all java files

Reply:
(please, make this modification in `ecocode` repository)

Reply:
Hi @dedece35, Okay I'll do it. Thanks.